### DISCUSSION TITLE
Include railways in lower zoom levels

### DISCUSSION THREAD

**Discussion by pietervdvn** [protomaps/basemaps#288] [Ideas]
Hi, as someone without a car who uses the train a lot, I'd love to have railways as high-level navigation (instead of only having major roads). However, railways are only included in z>=11, whereas major roads are included up to z>=5. Would it be possible to have (major) railways on a lower zoomlevel as well?

**Comment by wipfli** [comment]
Showing railways at lower zooms might be a good idea. You can probably simply edit the minzoom here: https://github.com/protomaps/basemaps/blob/0eeafa51cb02671615300e3433374eef66f1417a/tiles/src/main/java/com/protomaps/basemap/layers/Transit.java#L29

Do you know how to generate a tileset with planetiler yourself?

**Reply by pietervdvn** [reply]
Is this a 'please open a pull request' or a 'this won't be supported by the hosted/official version and build the files yourself'?

TBH, I'm not really fond on building the files myself as it takes a lot of work...

**Reply by nvkelso** [reply]
This could significantly increase tile size so caution is warranted but there's a path forward. We need to be cautious about increasing the tile size of anything tiles from zoom 0 to 10 especially. We'd need to vary the minZoom based on the type of railroad and what type of relations it's part of. 

There is prior art in Tilezen for this, see an old build here: 

- https://tangrams.github.io/bubble-wrap/#7/50.792/6.021 and enable "Transit" overlay in the right sidebar. 
- https://github.com/tilezen/vector-datasource/blob/master/yaml/transit.yaml#L22-L51

**Reply by wipfli** [reply]
I can help you make a tileset yourself. It is not that hard. For the railways we would only generate tiles for the railways at lower zoomlevels, so that is quite quick to make. Should I make an example project which has railways at z10?

**Reply by bdon** [reply]
Following tilezen I think we should attempt raising rail to 5+ and inspect the impact on tile size.

**Reply by pietervdvn** [reply]
> I can help you make a tileset yourself. It is not that hard. For the railways we would only generate tiles for the railways at lower zoomlevels, so that is quite quick to make. Should I make an example project which has railways at z10?

Thanks a lot for the offer to help, but it is simply that I don't have the time or energy to set this up, especialy because this is only a 'nice to have' for me.
